internal/scripts/docker/README.md

```diff
@@ -1,37 +1,71 @@
-# How to Use Dockerfiles
+How to Use Dockerfiles
+======================
 
-Run all of the Docker commands from the top-level directory of the MET repository
+GitHub actions are triggered for Pull Requests to the "develop" and "main_v*"
+branches and for pushes to the branches "feature_*", "bugfix_*", "develop",
+"develop-ref", and "main_v*" created in this repository to automatically build
+images from these Dockerfiles and push them to DockerHub repositories.  The
+MET Docker image is used in Dockerfiles in the METplus wrappers component
+repository for testing.
 
-## Build MET from clone
+All of the Docker commands below must be run from the top-level directory of
+the MET repository.
 
+1. `Dockerfile` checks out MET from GitHub and compiles the specified branch
+or tag from source, sets a working directory, downloads and installs MET and
+GhostScript fonts, and deletes the MET source code for tagged releases
+matching "v"*. It uses Docker image for the base compilation environment for
+MET from the
+[dtcenter/met-base](https://hub.docker.com/repository/docker/dtcenter/met-base)
+DockerHub repository.  Tagged versions are available in the
+[dtcenter/met](https://hub.docker.com/repository/docker/dtcenter/met-base)
+DockerHub repository.  This MET image can be built manually by running:
 ```
 docker build -t dtcenter/met:${TAG_NAME} --build-arg SOURCE_BRANCH=${BRANCH_NAME} internal/scripts/docker
 docker push dtcenter/met:${TAG_NAME}
 ```
 
 where:
-* TAG_NAME is the name of the DockerHub tag to create
-* BRANCH_NAME is the MET branch to checkout
-
-## Build MET from local source code with minimum requirements
+* `${TAG_NAME}` is the name of the DockerHub tag to create
+* `${BRANCH_NAME}` is the MET branch to checkout
 
+2. `Dockerfile.copy` compiles MET using the specified branch or tag from local
+source code, sets a working directory, downloads and installs MET and
+GhostScript fonts, and deletes the MET source code for tagged releases
+matching "v"*. It uses a local Docker image for the base compilation
+environment for MET from the
+[METbaseimage](https://github.com/dtcenter/METbaseimage/)
+GitHub repository. See the
+[METbaseimage README.md](https://github.com/dtcenter/METbaseimage/blob/main/README.md)
+file for more information on manually creating the Docker image for the base
+compilation environment. This MET image can be built manually by running:
 ```
 docker build -t dtcenter/met:${TAG_NAME} --build-arg SOURCE_BRANCH=${BRANCH_NAME} -f internal/scripts/docker/Dockerfile.copy .
 docker push dtcenter/met:${TAG_NAME}
 ```
 
 where:
-* TAG_NAME is the name of the DockerHub tag to create
-* BRANCH_NAME is the identifier to use for $MET_GIT_NAME inside image
-
-## Build MET from local source code with unit test requirements
+* `${TAG_NAME}` is the name of the DockerHub tag to create
+* `${BRANCH_NAME}` is the identifier to use for $MET_GIT_NAME inside image
 
+3. `Dockerfile.copy` compiles MET using the specified branch or tag from local
```

Comment:
You've listed `Dockerfile.copy` twice in the same list.

I'd recommend reorganizing the info in some way to avoid listing the same thing twice.

Reply:
Great point, @JohnHalleyGotway!  We definitely don't want to do that.  I am working on a rewrite now and will request a re-review once I am finished.  Thanks!